### PR TITLE
Update observability term

### PR DESCRIPTION
**Pull request by electrocucaracha**
### Describe your changes
This change synchronizes the changes made on the `observability` term to the Spanish version.

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by krol3 — APPROVED** [review]
LGTM

File: content/es/observability.md [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+## ¿Qué es?
 
-La observabilidad de un sistema tiene un impacto significativo en su costo operativo. Los sistemas observables brindan datos útiles y procesables a sus operadores, lo que les permite lograr resultados favorables y menos tiempo de inactividad. Tenga en cuenta que más información no se traduce necesariamente en un sistema más observable. De hecho, a veces la cantidad de información generada por un sistema puede dificultar la identificación de señales de salud valiosas desde el ruido generado por la aplicación. La observabilidad requiere los datos correctos para tomar las decisiones correctas.
+La observabilidad es la capacidad de generar y descubrir continuamente información procesable basada en señales del sistema bajo observación.
+En otras palabras, la observabilidad permite a los usuarios comprender el estado de un sistema a partir de su salida externa y tomar medidas (correctivas).
```
Comment (92nqb) [inline content/es/observability.md:11]:
```suggestion
En otras palabras, la observabilidad permite a los usuarios comprender el estado de un sistema a partir de su salida externa y así tomar medidas (correctivas).
```

File: content/es/observability.md [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+## Problema que aborda
+
+Los sistemas informáticos se miden mediante la observación de señales de bajo nivel, como el tiempo de CPU, la memoria, el espacio en disco y las señales comerciales y de nivel superior, incluidos los tiempos de respuesta de API, errores, transacciones por segundo, etc.
+
+La observabilidad de un sistema tiene un impacto significativo en su costo operativo.
+Los sistemas observables brindan datos significativos y procesables a sus operadores, lo que les permite lograr resultados favorables (respuesta más rápida a incidentes, mayor productividad del desarrollador) y menos trabajo y tiempo de inactividad.
+
+## ¿Cómo ayuda?
+
+Entendiendo de que más información no necesariamente significa que en un sistema sea más observable es esencial.
```
Comment (92nqb) [inline content/es/observability.md:22]:
```suggestion
Es esencial entender que más información no necesariamente significa que en un sistema sea más observable.
```

File: content/es/observability.md [hunk 3]
```diff
@@ -1,9 +1,24 @@
...
+De hecho, a veces, la cantidad de información generada por un sistema puede dificultar la identificación de señales de salud valiosas a partir del ruido generado por la aplicación.
```
Comment (92nqb) [inline content/es/observability.md:23]:
```suggestion
De hecho, a veces, la cantidad de información generada por un sistema puede dificultar la identificación de señales de salud valiosas debido al ruido generado por la aplicación.
```

**Review by 92nqb — CHANGES_REQUESTED** [review]
I've added some comments

**Review by krol3 — APPROVED** [review]
LGTM

**Review by 92nqb — APPROVED** [review]
LGTM

**Pull request merged**